packages/@expo/cli/src/api/endpoint.ts

```diff
@@ -10,3 +11,25 @@ export function getExpoApiBaseUrl(): string {
     return `https://api.expo.dev`;
   }
 }
+
+/** Get the URL for the expo.dev website. */
+export function getExpoWebsiteBaseUrl(): string {
```

Comment:
Any relation to https://github.com/expo/expo/blob/52d6405570a39a87149648d045d91098374f4423/packages/%40expo/cli/src/api/endpoint.ts#L4-L12 ?

If not, do these URLs interop for testing, i.e. would it be confusing if someone used "EXPO_LOCAL" and some API endpoints went to `http://expo.test` and others went to `http://127.0.0.1:3000`?

Reply:
http://127.0.0.1:3000 should be the same as api.expo.test and  http://127.0.0.1:3001 the same as expo.test, so these should be interoperable.

The SSO changes are the only thing using getExpoWebsiteBaseUrl() and these values for the URLs worked fine in manual testing of SSO .